openbook_auth/views/users/serializers.py

```diff
@@ -104,6 +104,7 @@ class GetUserUserSerializer(serializers.ModelSerializer):
     following_count = FollowingCountField()
     posts_count = PostsCountField()
     is_following = IsFollowingField()
+    is_follower = IsFollowerField()
     is_connected = IsConnectedField()
     is_fully_connected = IsFullyConnectedField()
     connected_circles = ConnectedCirclesField(circle_serializer=GetUserUserCircleSerializer)
```

Comment:
Just one comment 🤔  
The fields here are with respective to the logged in user
so `is_connected` ie is the logged in user connected to the user whose profile is being retrieved.
`is_reported` by is reported by logged in user
`is_following`  is the logged in user following the retrieved user
similarly .. the this field should follow that pattern and be called `is_followed` 
`is_followed` i.e  is the logged in user followed by the retrieved user.